easybuild/easyconfigs/x/XDS/XDS-3Nov2014-INTEL64_Linux_x86_64.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'XDS'
+version = '3Nov2014'
+versionsuffix = '-INTEL64_Linux_x86_64'
+
+homepage = 'http://xds.mpimf-heidelberg.mpg.de/'
+description = """ X-ray Detector Software for processing single-crystal
+ monochromatic diffraction data recorded by the rotation method. For a
+ description of the package, including references, visit xds_doc. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# source_urls = ['ftp://ftp.mpimf-heidelberg.mpg.de/pub/kabsch']
+# Download does not provide version information. Downloaded manually
+# and renamed with the date as version (check the main website for current
+# version number e.g. "3Nov2014")
```

Comment:
Recommendation: 
Please, use YYYYMMDD as date format, since that allows a little bit of automatic ordering to take place.
ie. we are looking at this: http://en.wikipedia.org/wiki/ISO_8601#Calendar_dates